lib/Elastica/Transport/AbstractTransport.php

```diff
@@ -99,14 +99,13 @@ public static function create($transport, Connection $connection, array $params
             } else {
                 $transport = ucfirst($transport);
             }
-
-            $className = 'Elastica\\Transport\\'.$transport;
-
-            if (!class_exists($className)) {
-                throw new InvalidException('Invalid transport');
+            $classNames = ["Elastica\\Transport\\$transport", $transport];
```

Comment:
[Check namespace seperator](https://github.com/ruflin/Elastica/pull/1169#discussion_r77565586) is bad idea, We should not assume that class has a namespace.